### PR TITLE
Add Logout Button

### PR DESCRIPTION
**Pull request by LubnaAly**

### CODE REVIEW

File: AuthenticationApp/Controller/LoginVC.swift [hunk 1]
```diff
@@ -22,17 +22,17 @@ class SignInVC: UIViewController {
     }
     
     @IBAction func signInButtonTapped(_ sender: Any) {
-        guard isValidUserData() else { return }
+//        guard isValidUserData() else { return }
```
Comment (MostafaNafie) [inline AuthenticationApp/Controller/LoginVC.swift]: Please uncomment this line.

File: AuthenticationApp/Controller/LoginVC.swift [hunk 2]
```diff
@@ -22,17 +22,17 @@ class SignInVC: UIViewController {
...
 
-private extension SignInVC {
+private extension LoginVC {
     func setupUI() {
-        title = "Sign In"
+        title = "Login"
         setupLabels()
         setupTextFields()
-        signInButtonTapped.setTitle("Sign In", for: .normal)
+        signInButtonTapped.setTitle("Login", for: .normal)
```
Comment (MostafaNafie) [inline AuthenticationApp/Controller/LoginVC.swift]: Please rename the outlets and actions to `login`.
```suggestion
        loginButtonTapped.setTitle("Login", for: .normal)
```

File: AuthenticationApp/Controller/ProfileVC.swift [hunk 3]
```diff
@@ -14,6 +14,8 @@ class ProfileVC: UIViewController {
     @IBOutlet weak var UserNameLabel: UILabel!
     @IBOutlet weak var UserEmailLabel: UILabel!
     @IBOutlet weak var UserGenderLabel: UILabel!
+    @IBOutlet weak var logOutButtonTapped: UIButton!
+
```
Comment (MostafaNafie) [inline AuthenticationApp/Controller/ProfileVC.swift]: Please remove the extra blank line.

File: AuthenticationApp/Controller/SignUpVC.swift [hunk 4]
```diff
@@ -117,7 +117,7 @@ private extension SignUpVC {
     }
     
     func goToSignIn() {
-        if let signInVC = storyboard?.instantiateViewController(identifier: "SignInVC") as? SignInVC {
+        if let signInVC = storyboard?.instantiateViewController(identifier: "SignInVC") as? LoginVC {
```
Comment (MostafaNafie) [inline AuthenticationApp/Controller/SignUpVC.swift]:
```suggestion
        if let loginVC = storyboard?.instantiateViewController(identifier: "LoginVC") as? LoginVC {
```

File: AuthenticationApp/Controller/ProfileVC.swift [hunk 5]
```diff
@@ -22,6 +24,12 @@ class ProfileVC: UIViewController {
         setupUI()
         setUserData()
     }
+        
+    @IBAction func logOutButtonTapped(_ sender: Any) {
+        if let signUpVC = storyboard?.instantiateViewController(identifier: "SignUpVC") as? SignUpVC {
```
Comment (MostafaNafie) [inline AuthenticationApp/Controller/ProfileVC.swift]: Please revisit this logic.

**Review by MostafaNafie — CHANGES_REQUESTED** [review]

File: AuthenticationApp/Controller/LoginVC.swift (hunk 1 above)
Comment (LubnaAly) [reply AuthenticationApp/Controller/LoginVC.swift]: Done.

File: AuthenticationApp/Controller/LoginVC.swift (hunk 2 above)
Comment (LubnaAly) [reply AuthenticationApp/Controller/LoginVC.swift]: Done.

File: AuthenticationApp/Controller/ProfileVC.swift (hunk 3 above)
Comment (LubnaAly) [reply AuthenticationApp/Controller/ProfileVC.swift]: Done.

File: AuthenticationApp/Controller/SignUpVC.swift (hunk 4 above)
Comment (LubnaAly) [reply AuthenticationApp/Controller/SignUpVC.swift]: Done

File: AuthenticationApp/Controller/ProfileVC.swift (hunk 5 above)
Comment (LubnaAly) [reply AuthenticationApp/Controller/ProfileVC.swift]: Done

**Review by MostafaNafie — APPROVED** [review]
Great job!! 🚀🚀

**Review by MostafaNafie — APPROVED** [review]

**Pull request merged**